src/include/cell.i

```diff
@@ -170,16 +171,58 @@ struct __wt_cell_unpack {
 	uint8_t ovfl;			/* boolean: cell is an overflow */
 };
 
+/*
+ * __cell_pack_timestamp_value --
+ *	Pack a start, stop timestamp pair for a value.
+ */
+static inline void
+__cell_pack_timestamp_value(
+    uint8_t **pp, wt_timestamp_t start_ts, wt_timestamp_t stop_ts)
+{
+	WT_ASSERT(NULL, start_ts <= stop_ts);
+
+	if (__wt_process.page_version_ts) {
+		/* Start timestamp, stop timestamp difference. */
+		(void)__wt_vpack_uint(pp, 0, start_ts);
+		(void)__wt_vpack_uint(pp, 0, stop_ts - start_ts);
+	}
+}
+
+/*
+ * __cell_pack_timestamp_addr --
+ *	Pack a oldest_start, newest_start, newest_stop timestamp triplet for an
+ * address.
+ */
+static inline void
+__cell_pack_timestamp_addr(uint8_t **pp, wt_timestamp_t oldest_start_ts,
+    wt_timestamp_t newest_start_ts, wt_timestamp_t newest_stop_ts)
+{
+	WT_ASSERT(NULL, oldest_start_ts <= newest_start_ts);
+	WT_ASSERT(NULL, newest_start_ts <= newest_stop_ts);
+
+	if (__wt_process.page_version_ts) {
+		(void)__wt_vpack_uint(pp, 0, oldest_start_ts);
+
+		/* Newest start timestamp, stop timestamp difference. */
+		(void)__wt_vpack_uint(pp, 0, newest_start_ts);
+		(void)__wt_vpack_uint(pp, 0, newest_stop_ts - newest_start_ts);
+	}
+}
+
 /*
  * __wt_cell_pack_addr --
  *	Pack an address cell.
  */
 static inline size_t
-__wt_cell_pack_addr(WT_CELL *cell, u_int cell_type, uint64_t recno, size_t size)
+__wt_cell_pack_addr(WT_CELL *cell, u_int cell_type, uint64_t recno,
```

Comment:
Not really? They won't fit on the same line, so the second line has to be broken somewhere, I'm happy to switch to something else if it reads more naturally for other people.

Were you thinking it should be:
```
static inline size_t
__wt_cell_pack_addr(WT_CELL *cell, u_int cell_type, uint64_t recno,
    wt_timestamp_t oldest_start_ts, wt_timestamp_t newest_start_ts, 
    wt_timestamp_t newest_stop_ts, size_t size)
```
instead?

Reply:
Agree that having `wt_timestamp_t oldest_start_ts` on a line on its own seems odd, but since we don't have hard and fast rules for this, I'm happy for someone to try something different next time they're in the area.